data/mapcss-mapping.csv

```diff
@@ -395,8 +395,7 @@ place|farm;294;
 deprecated:boundary|administrative|11:04.2024;[boundary=administrative][admin_level=11];x;name;int_name;295;
 waterway|weir;296;
 highway|secondary|tunnel;[highway=secondary][tunnel?];;name;int_name;297;
-# TODO: its a mismapping likely, if there is a bridge structure still, then it should be railway=disused.
-railway|abandoned|bridge;[railway=abandoned][bridge?];;name;int_name;298;
+railway|disused|bridge;[railway=disused][bridge?];;name;int_name;298;
```

Comment:
actually a `railway=abandoned bridge=yes` is not a mismapping, its a legit situation, e.g. rails were removed but bridge is still there with some sleepers, posts and maybe other remnants left...
according to https://taginfo.openstreetmap.org/tags/railway=abandoned#combinations there are 37k abandoned railway bridges mapped (9% of all abandoned railways)

Reply:
Ideally we'd have separate rendering styles for abandoned and disused bridges which would be similar to abandoned and disused lines respectively. Then we'll need to add translations into `types_strings.txt` also.

If you feel its too much to do then I think its fine to combine both of them into a single style, e.g.
`railway|abandoned|bridge;[railway=abandoned][bridge?],[railway=disused][bridge?];;name;int_name;298;`.
And they would use existing translations for `railway|abandoned|bridge`.